src/migrate/transforms/rename-jsx-import.test.js

```diff
@@ -0,0 +1,128 @@
+const { dedent } = require('ts-dedent');
+const { InMemoryFileSystemHost, Project } = require('ts-morph');
+
+const transform = require('./rename-jsx-import');
+
+describe('transform', () => {
```

Comment:
Can the test be in TypeScript, even though the source is JS? Would be nice to still get type checking at least for the tests.